transactions/update-contract/2023/feb-23/arguments-update-contract-FlowIDTableStaking-mainnet.json

```diff
@@ -2,6 +2,6 @@
6e735374616b656442794e6f646554797065203d207b55496e74382831293a20302e302c2055496e74382832293a20302e302c2055496e74382833293a20302e302c2055496e74382834293a20302e302c2055496e74382835293a20302e307d0a202020202020202073656c662e65706f6368546f6b656e5061796f7574203d2065706f6368546f6b656e5061796f75740a202020202020202073656c662e6e6f646544656c65676174696e67526577617264437574203d207265776172644375740a202020202020202073656c662e726577617264526174696f73203d207b55496e74382831293a20302e3136382c2055496e74382832293a20302e3531382c2055496e74382833293a20302e3037382c2055496e74382834293a20302e3233362c2055496e74382835293a20302e307d0a0a20202020202020206c657420617070726f76654c6973743a207b537472696e673a20426f6f6c7d203d207b7d0a202020202020202073656c662e7365745061727469636970616e744e6f64654c69737428617070726f76654c697374290a202020202020202073656c662e6163636f756e742e736176653c7b537472696e673a20426f6f6c7d3e28617070726f76654c6973742c20746f3a202f73746f726167652f69645461626c65417070726f76654c697374290a0a20202020202020206c6574206e6f6e4f7065726174696f6e616c4c6973743a207b537472696e673a205546697836347d203d207b7d0a202020202020202073656c662e6163636f756e742e736176653c7b537472696e673a205546697836347d3e286e6f6e4f7065726174696f6e616c4c6973742c20746f3a202f73746f726167652f69645461626c654e6f6e4f7065726174696f6e616c4e6f6465734c697374290a0a20202020202020206c6574206d6f76657350656e64696e674c6973743a207b537472696e673a207b55496e7433323a20426f6f6c7d7d203d207b7d0a202020202020202073656c662e6163636f756e742e736176653c7b537472696e673a207b55496e7433323a20426f6f6c7d7d3e286d6f76657350656e64696e674c6973742c20746f3a202f73746f726167652f69645461626c654d6f76657350656e64696e674c697374290a0a20202020202020206c657420656d70747943616e6469646174654e6f6465733a207b55496e74383a207b537472696e673a20426f6f6c7d7d203d207b313a207b7d2c20323a207b7d2c20333a207b7d2c20343a207b7d2c20353a207b7d7d0a2020202020202020466c6f7749445461626c655374616b696e672e6163636f756e742e7361766528656d70747943616e6469646174654e6f6465732c20746f3a202f73746f726167652f69645461626c6543616e6469646174654e6f646573290a0a20202020202020202f2f2053617665207468652063616e646964617465206e6f646573206c696d69740a2020202020202020466c6f7749445461626c655374616b696e672e6163636f756e742e736176653c7b55496e74383a2055496e7436347d3e2863616e6469646174654e6f64654c696d6974732c20746f3a202f73746f726167652f69645461626c6543616e6469646174654e6f64654c696d697473290a0a20202020202020206c657420736c6f744c696d6974733a207b55496e74383a2055496e7431367d203d207b313a2031303030302c20323a2031303030302c20333a2031303030302c20343a2031303030302c20353a2031303030307d0a20202020202020202f2f205361766520736c6f74206c696d6974730a2020202020202020466c6f7749445461626c655374616b696e672e6163636f756e742e7361766528736c6f744c696d6974732c20746f3a202f73746f726167652f666c6f775374616b696e67536c6f744c696d697473290a0a20202020202020206c657420736c6f74436f756e74733a207b55496e74383a2055496e7431367d203d207b313a20302c20323a20302c20333a20302c20343a20302c20353a20307d0a20202020202020202f2f205361766520736c6f7420636f756e74730a2020202020202020466"
```

Comment:
this hex representation is incomplete and gets clipped at the end when you try to convert it back to ASCII. I have regenerated the hex value of the CDC